rskj-core/src/test/java/co/rsk/peg/BridgeTest.java

```diff
@@ -2266,6 +2266,40 @@ void getProposedFederationAddress(MessageCall.MsgType msgType, ActivationConfig
         }
     }
 
+    @ParameterizedTest()
+    @MethodSource("msgTypesAndActivations")
+    void getProposedFederationSize(MessageCall.MsgType msgType, ActivationConfig activationConfig) throws VMException {
```

Comment:
Same comment as the others. Get rid of the conditional assertions.

Reply:
I agree.